testing/run_tests.py

```diff
@@ -498,7 +499,7 @@ def make_test(name, flags=None, extra_env=None):
         build_dir,
         'impeller_unittests',
         executable_filter,
-        shuffle_flags,
+        ['--gtest_filter=-*/Vulkan'] + shuffle_flags,
```

Comment:
Now that the bot is building with vulkan enabled we need to turn these tests off.  I tried to run them with swiftshader and it didn't work.  I didn't look into it closely, opting to turn them off instead.  It doesn't seem worthwhile to make this a flag.